geniza/templates/nav.html

```diff
@@ -1,17 +1,100 @@
+                <a href="#" role="menuitem">{% translate "Contact Us" %}</a>
+            </li>
+            <li class="menu-item has-submenu" role="none">
+                {# Translators: label for 'about' link in main navigation #}
+                {% translate "About" as about_label %}
+                <a id="open-about-menu"
+                    href="#about-menu"
+                    aria-haspopup="true"
+                    aria-expanded="false"
+                    role="menuitem">
+                    {{ about_label }}
+                </a>
+                {# Translators: accessibility label for 'about' menu in navigation #}
+                {% translate "about menu" as about_menu_label %}
+                <ul id="about-menu" role="menu" aria-label="{{ about_menu_label }}">
+                    <li class="menu-button" role="none">
+                        <a id="back-to-main-menu" href="#menu" role="button">
+                            {# Translators: label for main menu back button for mobile navigation #}
+                            {% translate "Return to main menu" as back_button %}
+                            <span class="sr-only">{{ back_button }}</span>
+                        </a>
+                    </li>
+                    <li class="menu-label" role="none">
+                        <span>{{ about_label }}</span>
+                    </li>
+                    <li class="menu-button" role="none">
+                        <a href="#" role="button">
+                            <span class="sr-only">{{ close_button }}</span>
+                        </a>
+                    </li>
+                    <li class="menu-item" role="none">
+                        {# Translators: label for link to 'Credits' page in main navigation #}
+                        <a href="#" role="menuitem">{% translate "Credits" %}</a>
+                    </li>
+                    <li class="menu-item" role="none">
+                        {# Translators: label for link to 'How to Cite' page in main navigation #}
+                        <a href="#" role="menuitem">{% translate "How to Cite" %}</a>
+                    </li>
+                    <li class="menu-item" role="none">
+                        {# Translators: label for link to 'Data Exports' page in main navigation #}
+                        <a href="#" role="menuitem">{% translate "Data Exports" %}</a>
+                    </li>
+                    <li class="menu-item" role="none">
+                        {# Translators: label for link to 'Technical' page in main navigation #}
+                        <a href="#" role="menuitem">{% translate "Technical" %}</a>
+                    </li>
+                    <li class="menu-item" role="none">
+                        {# Translators: label for link to 'FAQ' page in main navigation #}
+                        <a href="#" role="menuitem">{% translate "FAQ" %}</a>
+                    </li>
+                </ul>
```

Comment:
I don't know if these should use the translate tag or if we want to use wagtail's i18n — probably we'll need the wagtail functionality (was just reading up, it looks pretty powerful https://docs.wagtail.io/en/stable/advanced_topics/i18n.html#wagtail-s-approach-to-multi-lingual-content ) — but we may not have that for MVP so in the meantime this probably makes sense.